composer.json

```diff
@@ -10,6 +10,7 @@
     "html",
     "newsletters"
   ],
+  "version": "7.0.0",
```

Comment:
version should not be hardcoded, it is inferred from git tag.

Reply:
Ok, I never understood that till this easy explanation ;-)